src/table/src/table.rs

```diff
@@ -28,6 +28,8 @@ use crate::error::Result;
 use crate::metadata::{FilterPushDownType, TableId, TableInfoRef, TableType};
 use crate::requests::{AlterTableRequest, InsertRequest};
 
+pub type AlterContext = anymap::Map<dyn Any + Send + Sync>;
```

Comment:
Define an empty struct instead of anymap like `WriteContext`?
https://github.com/GreptimeTeam/greptimedb/blob/8ffc078f88f5e265eb0821820593a1c4bd309c17/src/store-api/src/storage/region.rs#L79

Reply:
I need to carry GRPC AlterExpr in context, something not able to be defined as concrete type in struct in table crate. Otherwise table crate would rely on api.